_docs/sysadmin/version_notes/v20.07.00.md

```diff
@@ -0,0 +1,26 @@
+---
+title:  v20.07.00 -- Additional WebSockets Support
+---
+
+To support the expanded use of Websockets for additional dynamic pages
+within the Submitty website, we must remove a restriction on a
+specific php function.
+
+Version
+[v20.07.00](https://github.com/Submitty/Submitty/releases/v20.07.00)
+requires the following edits:
+
+1. Locate your `php.ini` file.  
+   _It is probably something like `/etc/php/7.2/fpm/php.ini`._
+
+
+2. Edit this file:  
+   Remove `stream_socket_client` from the list of `disabled_functions`.  
+   Save the file.
+
+
+3. Restart php fpm:
+
+   ```
+   sudo systemctl restart php7.2-fpm
```

Comment:
In the interest of total portability to other distros that might be running a different PHP version:
```suggestion
   sudo systemctl restart php$(php -r 'print PHP_MAJOR_VERSION.".".PHP_MINOR_VERSION;')-fpm
```